app/api/services/users/create_user.py

```diff
@@ -0,0 +1,31 @@
+from uuid import uuid4
+
+from api.db.models.user_models import User, UserRole
+from api.route.api_context import ApiContext
+from api.route.models.user import UserParams, UserResponse
+
+
+def create_user(api_context: ApiContext) -> UserResponse:
+    request = UserParams.parse_obj(api_context.request_body)
+
+    user = User(
```

Comment:
I told Jesse not to touch the db classes for now since i was working on that and didn't want conflicts. we could always refine the interfaces in future PRs

Reply:
Very much agreed with all this! Trying to make forward progress in bite-sized PRs and not step on toes. Per Loren's other comment, I should spell out some high-level goals for this work!